stable/metrics-server/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.3.1
 description: Metrics Server is a cluster-wide aggregator of resource usage data.
 name: metrics-server
-version: 2.1.0
+version: 2.1.1
```

Comment:
This needs a minor version bump

Reply:
In the appVersion?

Reply:
No, the chart version. Introducting new features requires minor version bump. patch version is only for bugfixes and similar